specifications/sep-payment-channel-transactions.md

```diff
@@ -612,8 +614,9 @@ check if either participant has clawback enabled. Checking the state after
 formation is critical because there is no way for participants to guarantee
 trustline state until after formation has completed because the state can change
 prior to formation. For this reason participants should perform their initial
-deposit after formation, unless they trust the asset issuer not to clawback from
-payment channel escrow accounts, or unless the asset is auth immutable.
+deposit after formation, unless they have no requirement for the asset not to be
+clawback enabled, or they trust the asset issuer not to clawback from payment
```

Comment:
i think. If not, the double negative is hard to digest
```suggestion
deposit after formation, unless they have no requirement for the asset to be
clawback enabled, or they trust the asset issuer not to clawback from payment
```

Reply:
changing the suggestion based off your explanation above. But if this is not the specific intent your trying to say then nvrm 

```suggestion
deposit after formation, unless they are okay with the asset potentially being clawback enabled, or they trust the asset issuer not to clawback from payment
```